tilesets/scripts/utils.py

```diff
@@ -25,68 +28,76 @@ def flatten(files):
 
 def print_response(text):
     try:
-      j = json.loads(text)
-      msg = json.dumps(j, indent=2, sort_keys=True)
-      click.echo(msg)
-    except:
-      click.echo('Failure \n' + text)
+        j = json.loads(text)
+        msg = json.dumps(j, indent=2, sort_keys=True)
+        click.echo(msg)
+    except:  # tofix: bare except
+        click.echo("Failure \n" + text)
+
+
+def validate_tileset_id(tileset_id):
```

Comment:
Added this for two reasons:
1. I couldn't ignore the rule that this replaced (https://github.com/mapbox/tilesets-cli/blob/ea900c3ad490eb4b111b670c459d3015f0d6a1fa/tilesets/cli.py#L64) without allowing a few other potential bad things
2. Docstring example! We could also add docstring verification to the pre-commit hooks.